### DISCUSSION TITLE
panel model vs obs: stats, timeseries AND map

### DISCUSSION THREAD

**Discussion by jmccreight** [hytest-org/hytest#308] [Model Evaluation Methods]
@rsignell-usgs and other panel users, 

I've build a holoviz/panel function to view the comparison of spatially distributed timeseries like model v obs at a variety of streamflow locations. Here is a screen shot

![image](https://github.com/hytest-org/hytest/assets/12465248/9288a614-2b4e-4d3c-89cf-a387a58485d7)

It is interactive: you click on a point within violin plot and it opens that locations timeseries (modeled and obs). I'd really like to integrate a map with that but havent found any good examples that I would start from (though it's been a while since I looked). The desired functionality would be to change the size & color of the selected location (from the boxplot) on the map so that one could identify it. It would also be desirable to select locations from the map and have these appear in the timeseries.

Thanks for your suggestions. 
James

**Comment by rsignell-usgs** [comment]
@alaws-USGS I was hoping you would have some thoughts here!

**Reply by sfoks** [reply]
.

**Comment by sfoks** [comment]
Nice plots! Gene (@gzt5142) has an interactive point-n-click map with timeseries data that could be a useful example for you? 

Example 4 in:
https://github.com/hytest-org/training-2023-model-evaluation/blob/main/labs/02a-gt-holoviz-demo.ipynb
